src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/resources/MigrationHistoryResource.kt

```diff
@@ -1,117 +0,0 @@
-package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.resources
-
-import io.swagger.v3.oas.annotations.Operation
-import io.swagger.v3.oas.annotations.Parameter
-import io.swagger.v3.oas.annotations.media.ArraySchema
-import io.swagger.v3.oas.annotations.media.Content
-import io.swagger.v3.oas.annotations.media.Schema
-import io.swagger.v3.oas.annotations.responses.ApiResponse
-import org.springframework.format.annotation.DateTimeFormat
-import org.springframework.http.HttpStatus
-import org.springframework.http.MediaType
-import org.springframework.security.access.prepost.PreAuthorize
-import org.springframework.web.bind.annotation.DeleteMapping
-import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.RequestParam
-import org.springframework.web.bind.annotation.ResponseStatus
-import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.config.ErrorResponse
-import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.persistence.repository.MigrationHistory
-import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.HistoryFilter
-import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.service.MigrationHistoryService
-import java.time.LocalDateTime
-
-@RestController
-@RequestMapping("/", produces = [MediaType.APPLICATION_JSON_VALUE])
-class MigrationHistoryResource(private val migrationHistoryService: MigrationHistoryService) {
-  @PreAuthorize("hasRole('ROLE_MIGRATION_ADMIN')")
```

Comment:
these two look like they were only added for the tests - have called the repository instead to `deleteAll`.